src/main/java/ch/heigvd/dai/tcp/Client.java

```diff
@@ -55,10 +55,23 @@ public void list(BufferedReader in, BufferedWriter out, Path path) throws IOExce
 
   }
 
+  public void delete(BufferedReader in, BufferedWriter out, Path path) throws IOException {
+    out.write("DELETE " + path + Server.NEW_LINE);
+    out.flush();
+
+    int status = Character.getNumericValue(in.read());
+    // remove \n or EOT chars
+    in.read();
+    if (status != 0) {
+      System.err.println("Got error: " + Errno.getErrorMessage(status));
+      return;
+    }
+
+    // TODO: process result ?
```

Comment:
You should only inform the user about the status result from the server.

Reply:
replaced todo with a println